metaflow/plugins/metadata/service.py

```diff
@@ -17,6 +20,18 @@ def __init__(self, msg, http_code=None, body=None):
         super(ServiceException, self).__init__(msg)
 
 
+def default_request_args():
+    return {"headers": METADATA_SERVICE_HEADERS, "auth": auth}
+
+
+def http_get(uri, **kwargs):
```

Comment:
Can you add a brief docstring to explain the purpose (to make sure the default args are added)